serbian/burekSir/pocetak/pocetak.md

```diff
@@ -0,0 +1,28 @@
+Čemu bes, čemu ozlojađenost. Pekara može i sme da ostane bez mog omiljenog bureka.
+Ljudi oko mene mogu da se smeju. Loše stvari mogu da se dese. Dobre stvari mogu da se dese. 
+Moje mišljenje i osećanje da nešto mora da se desi je skroz pogrešno:
+
+ "Ova pekara MORA da ima burek s mesom".
+
+Koja sam ja budala. Ne mora! Može da se rasproda, može pekar da zaboravi da napravi dovoljnu količinu,
+ može neko da kupi 5 kila bureka za proslavu...
+
+Svet dozvoljava sve mogućnosti. Zašto sam ovako limitirao svoje mišljenje???
+Pa zar me toliko bole noge da ne mogu da odem u drugu pekaru? :) Ko da ne mogu kupim nešto drugo za jelo :)
+
+Sećam se kako je govorio onaj matori:
+
+	Ljudski životi su prepuni teških nesreća.
+	Od kojih se većina nikad nije desila.
+
+	
+Da. Moj život je pun strahovitih nesreća. A najveća je što ova pekara nema burek s mesom. Hahahahaha :)
+
+
+Tako je počela njegova [promena](promena/promena.md).
+
+[Idem da se napijem.](http://zenpencils.com/comic/78-jessie-b-rittenhouse-my-wage)

```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.